src/profiler/profiler.h

```diff
@@ -843,6 +854,8 @@ struct ProfileTask : public ProfileDuration {
   VTUNE_ONLY_CODE(std::unique_ptr<vtune::VTuneTask> vtune_task_);
   /*! \brief NVTX duration object */
   NVTX_ONLY_CODE(std::unique_ptr<nvtx::NVTXDuration> nvtx_duration_);
+  /*! \brief not to add this stat to AggregateStats */
```

Comment:
nit: 'not to add...' -> Add profiler stat to AggregateStats.

Reply:
fixed